.github/workflows/meshmodel-generators.yml

```diff
@@ -82,6 +84,8 @@ jobs:
             cd scripts/component_updater
             go build main.go
             ./main https://docs.google.com/spreadsheets/d/e/2PACX-1vSgOXuiqbhUgtC9oNbJlz9PYpOEaFVoGNUFMIk4NZciFfQv1ewZg8ahdrWHKI79GkKK9TbmnZx8CqIe/pub\?gid\=0\&single\=true\&output\=csv --system meshery ../../server/meshmodel
+      - name: Pull changes from remote
```

Comment:
Shouldn’t this be before running the updater?
Otherwise the changes in the file due to component updates would conflict with the chnages from remote.